js/src/main/scala/java/util/TimeZone.scala

```diff
@@ -82,9 +90,34 @@ abstract class TimeZone extends Serializable with Cloneable {
   }
 
   def getDisplayName(daylight: Boolean, style: Int, locale: Locale): String = {
-    if (style != TimeZone.SHORT || style != TimeZone.LONG)
+    if (style != TimeZone.SHORT && style != TimeZone.LONG)
       throw new IllegalArgumentException(s"Illegal timezone style: $style")
-    ???
+
+    // Safely looks up given index in the array
+    def atIndex(strs: Array[String], idx: Int): Option[String] = {
+      if (idx >= 0 && idx < strs.length) Option(strs(idx))
+      else None
+    }
+
+    val id = getID
+    def currentIdStrings(strs: Array[String]): Boolean =
+      atIndex(strs, 0).contains(id)
```

Comment:
This is failing on travis. It seems `contains` is not supported on 2.10. Could you change this to e.g. `exists`?